### PR TITLE
Feature/cax 1 prs@86 load test data from tdg

### PR DESCRIPTION
**Pull request by Izzzu**
Load test data from the TDM data generator into the PRS database.

Added steps to the CD pipeline, and a script to run the TDM generator, convert the JSON output into SQL COPY commands and load the data into PostgreSQL.

Note: PR is in progress, missing:
- purging PRS database before copying data into it
- using TDM url of deployed instance instead of localhost

### CODE REVIEW

File: .github/workflows/partsrelationshipservice.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: PRS
+name: PRS CI
```
Comment (marcgs) [inline .github/workflows/partsrelationshipservice.yml:1]: This renaming will break https://github.com/catenax/tractusx/blob/main/.github/workflows/partsrelationshipservice-smoketest.yml#L5

File: .github/workflows/partsrelationshipservice-deploy.yml [hunk 2]
```diff
@@ -29,21 +30,38 @@ jobs:
...
-          terraform apply -auto-approve
+          pushd terraform
+            terraform init
+            terraform apply -auto-approve
+            terraform output -json > terraform-outputs.json
 
         env:
           AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
 
+      - name: "Load test data"
```
Comment (marcgs) [inline .github/workflows/partsrelationshipservice-deploy.yml:53]: Should we have this as a separate workflow? IMO it makes no sense to load data only on deployment, but rather every time that we are informed there is changes or regularly on a schedule.

File: .github/workflows/partsrelationshipservice-deploy.yml [hunk 3]
```diff
@@ -29,21 +30,38 @@ jobs:
...
+          pushd terraform
+            export POSTGRES_USER=$(jq -r .prs_db_administrator_username.value terraform-outputs.json)
+            export POSTGRES_PASSWORD=$(jq -r .prs_db_administrator_login_password.value terraform-outputs.json)
+            export POSTGRES_HOST=$(jq -r .prs_db_fqdn.value terraform-outputs.json)
+            export POSTGRES_DB=$(jq -r .prs_db_name.value terraform-outputs.json)
+          popd
+
+          pushd cd/test-data
+            # RUN --mount=type=cache is used in the tdm generator Dockerfile and the --mount option requires BuildKit.
+            DOCKER_BUILDKIT=1 ./load-test-data.sh
```
Comment (marcgs) [inline .github/workflows/partsrelationshipservice-deploy.yml:67]: Another TODO here is to replace this script and use the Broker API once available to load data

File: .github/workflows/partsrelationshipservice-deploy.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: Deploy PRS
+name: PRS CD
```
Comment (Izzzu) [inline .github/workflows/partsrelationshipservice-deploy.yml:1]: Breaks trigger here: https://github.com/catenax/tractusx/blob/main/.github/workflows/partsrelationshipservice-smoketest.yml#L5

**Pull request closed without merging**